### ISSUE TITLE
childViewEvents not working when triggering from a child rendered in a region

### ISSUE THREAD

**Issue by posti85** [marionettejs/backbone.marionette#3316]
### Actual behavior

I've done a simple example to ilustrate the issue: when the button b1 is pressed, the parent view doesn't listen the event (the 'button1Pressed' isn't shown in the console). However, when the button b2 is pressed, it does.

```javascript
const LayoutView = Marionette.View.extend({

    template: _.template('<div class="region"></div>'),

    regions: {
        region: '.region'
    },

    childViewEvents: {
        button1Pressed: 'button1Pressed',
        button2Pressed: 'button2Pressed'
    },

    button1Pressed() {
        console.log('button1Pressed');
    },

    button2Pressed() {
        console.log('button2Pressed');
    }
});

const RegionView = Marionette.View.extend({

    template: _.template(
        `<button class="b1">Event</button>
         <button class="b2">Trigger</button>`),

    events: {
        'click .b1': 'button1Pressed'
    },

    triggers: {
        'click .b2': 'button2Pressed'
    },

    button1Pressed() {
        this.trigger('button1Pressed');
    }
});

const app = new Marionette.Application({
    region: '#main'
});

app.start();

var layoutView = new LayoutView();

var regionView = new RegionView();

app.showView(layoutView);

layoutView.showChildView('region', regionView);
```

### Expected behavior

It should work the same way form button b1 and button b2.

### Environment

1. Marionette version: 3.0.1
2. Backbone version: 1.3.3
3. Additional build tools, etc:

**Comment by rafde** [comment]
That's intentional.
http://marionettejs.com/docs/master/marionette.view.html#view-triggers
Why do you expect `events` and `triggers` to act the same way?

**Comment by paulfalgout** [comment]
Actually the issue he's describing is that `childViewEvents` on a Layout doesn't work with `trigger`   It _does_ however work with `triggerMethod`, but that has been fixed for v3.2 here: https://github.com/marionettejs/backbone.marionette/pull/3247

**Comment by posti85** [comment]
Yeah, that is! Thank you for the clarification and the fix!
